tests/phpunit/src/Commands/Api/ApiCommandTest.php

```diff
@@ -400,4 +400,60 @@ public function testGetApplicationUuidFromBltYml(): void {
     $this->fs->remove($bltConfigFilePath);
   }
 
+  /**
+   * Test of deletion of the user from organization by user uuid.
+   */
+  public function testOrganizationMemberDeleteByUserUuid(): void {
+    ClearCacheCommand::clearCaches();
```

Comment:
Is clearing caches absolutely necessary? Clearing caches is not thread safe, so if you really need this you need to add this test to the serial group. I think the annotation is `@group serial`. But in most cases, you don't need to clear caches as part of a test.

Reply:
removed!